README.md

```diff
@@ -21,6 +21,8 @@
 
 **nf-core/methylseq** is a bioinformatics analysis pipeline used for Methylation (Bisulfite) sequencing data. It pre-processes raw data from FastQ inputs, aligns the reads and performs extensive quality-control on the results.
 
+![image](https://github.com/user-attachments/assets/9bec0af6-1eda-4d1d-9452-9dfba136ba6e)
```

Comment:
Fix the link

Reply:
I thought it was autogenerated by github ?

Reply:
Probably from a drag and drop, but you want to point to the actual file

Reply:
Fixing now one sec